cypress/integration/LoggedOutStaticFAQ.js

```diff
@@ -5,7 +5,7 @@ describe('Display Static FAQ', () => {
   it('User can navigate into FAQ Article', () => {
     cy.visit('/');
     cy.get('[data-cy=btn-nonexistent-booking]').trigger('click');
-    cy.get('.signInOrBack div').contains('Sign In');
+    cy.get('.signInOrBack div a').should('exist');
```

Comment:
I would be more specific, by omitting `Sign In`, it's no longer clear what is expected to be there, `.signInOrBack` has multiple elements inside

Reply:
I tried to do it more specific using another class inside `.signInOrBack .desktopOnly a`, looking for `Sign In` is giving me this error, any idea? ![screenshot](https://i.gyazo.com/2062a299de5a89faa7f1873c449049bc.png)

Reply:
I see, weird, might be some whitespace, although I don't know why, otherwise it's 👍  now

Reply:
yes, it was stuff related to whitespace 🙂